pandas/tests/series/test_constructors.py

```diff
@@ -134,6 +134,23 @@ def test_constructor_no_data_index_order(self):
         result = pd.Series(index=['b', 'a', 'c'])
         assert result.index.tolist() == ['b', 'a', 'c']
 
+    def test_constructor_no_data_string_type(self):
+        # GH 22477
+        result = pd.Series(index=[1], dtype=str)
+        assert np.isnan(result.iloc[0])
+
+    @pytest.mark.parametrize('item', ['13'])
```

Comment:
you don't need to parameterize this test (only 1 case), and you need to change the name as the next test overwrites it.